pkg/code-generator/docgen/generator.go

```diff
@@ -147,22 +147,37 @@ func (d *DocsGen) FileHeader(filename string) string {
 		return ".. Code generated by solo-kit. DO NOT EDIT."
 	}
 	if d.DocsOptions.Output == options.Hugo {
-		name := filepath.Base(filename)
-		if strings.HasSuffix(name, d.protoSuffix()) {
-			name = name[:len(name)-len(d.protoSuffix())]
-		}
-		return fmt.Sprintf(`
+		return d.hugoFileHeader(filename)
+	}
+	return `<!-- Code generated by solo-kit. DO NOT EDIT. -->
+`
+}
+
+func (d *DocsGen) hugoFileHeader(filename string) string {
+	name := filepath.Base(filename)
+
+	var title string
+	if strings.HasSuffix(name, d.protoSuffix()) {
+		// Remove the "sk.md" extensions
+		name = name[:len(name)-len(d.protoSuffix())]
```

Comment:
Out of scope, but why not this?
```suggestion
		name = strings.TrimSuffix(name, d.protoSuffix())
```